astroplan/plots/__init__.py

```diff
@@ -1,2 +1,10 @@
+"""
+`astroplan.plots` is a module for making plots of important quantities within
+the `astroplan` toolbox, such as airmass vs. time, etc.
+
+Note that, at minimum, you must construct `Observer`, `Target` and
+`astropy.time.Time` objects as input for all plots.
```

Comment:
@eteq: Since there's no tutorials page yet, I'll add a link in to it in the other docs branch.